src/Identity/UI/src/Areas/Identity/Pages/V5/Account/AccessDenied.cshtml.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Microsoft.AspNetCore.Identity.UI.V4.Pages.Account.Internal
```

Comment:
I think we might want to change these to be V5 namespaces right @javiercn or will this change so it doesn't matter?